sdk/ml/azure-ai-ml/azure/ai/ml/entities/_feature_store/_constants.py

```diff
@@ -0,0 +1,9 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+OFFLINE_STORE_CONNECTION_NAME = "OfflineStoreConnectionName"
+OFFLINE_MATERIALIZATION_STORE_TYPE = "azure_data_lake_gen2"
+OFFLINE_STORE_CONNECTION_CATEGORY = "ADLSGen2"
+DEFAULT_SPARK_RUNTIME_VERSION = "3.1.0"
```

Comment:
can we make default to 3.2.0 ?

Reply:
Okay, will update in Yaml and here together.